lib/commands/run/index.js

```diff
@@ -0,0 +1,152 @@
+                    if (err) {
+                        Messenger.getInstance().error(err);
+                        return cb(err);
+                    }
+                    Messenger.getInstance()
+                        .info('\n*****Once the session is successfully started, '
+                            + 'you can use `ask dialog` to make simulation requests to your local skill code*****\n');
+                    if (cmd.waitForAttach) {
+                        Messenger.getInstance()
+                            .info(`\n*****Debugging session will wait until inspector is attached at port - ${debugPort}*****\n`);
+                    }
+                    runFlow.execCommand();
+                });
+        });
+    }
+
+    _getAccessTokenForProfile(profile, debug, callback) {
+        const authorizationController = new AuthorizationController({
+            auth_client_type: 'LWA',
+            doDebug: debug
+        });
+        authorizationController.tokenRefreshAndRead(profile, (tokenErr, token) => {
+            if (tokenErr) {
+                return callback(tokenErr);
+            }
+            callback(null, token);
+        });
+    }
+
+    _getHostedSkillRuntime(smapiClient, skillId, callback) {
+        smapiClient.skill.alexaHosted.getAlexaHostedSkillMetadata(skillId, (err, response) => {
+            if (err) {
+                return callback(err);
+            }
+            if (response.statusCode >= 300) {
+                const error = jsonView.toString(response.body);
+                return callback(error);
+            }
+            try {
+                const { runtime } = response.body.alexaHosted;
+                if (!stringUtils.isNonBlankString(runtime)) {
+                    throw new CliError(`Unable to determine runtime of the hosted skill - ${skillId}`);
+                }
+                callback(null, helper.getNormalisedRuntime(runtime));
+            } catch (error) {
+                return callback(error);
+            }
+        });
+    }
+
+    _getSkillRunFlow(skillId, profile, skillCodeRegion, waitForAttach, watch, debug, debugPort, token, runRegion, callback) {
+        if (this._filterAlexaHostedSkill(profile)) {
+            const smapiClient = new SmapiClient({
+                profile,
+                doDebug: debug
+            });
+            this._getHostedSkillRuntime(smapiClient, skillId, (err, runtime) => {
+                if (err) {
+                    return callback(err);
+                }
+                try {
+                    callback(null, helper.getSkillFlowInstance(runtime, helper.getHostedSkillInvocationInfo(runtime),
+                        waitForAttach, debugPort, token, skillId, runRegion, watch));
+                } catch (error) {
+                    return callback(error);
+                }
+            });
+        } else {
+            try {
+                const skillCodeFolderName = helper.getSkillCodeFolderName(profile, skillCodeRegion);
```

Comment:
could you use Messenger.info to display this code folder name? Let user know what codebase we are using as endpoint to run